Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml

```diff
@@ -299,6 +308,36 @@
   - type: ItemBorgModule
     items:
     - RCDRecharging
+    
+- type: entity
+  id: BorgModuleStun
+  parent: [ BaseBorgModuleSecurity, BaseProviderBorgModule ]
+  name: Disabler cyborg module
+  components:
+  - type: Sprite
+    layers:
+    - state: security
+    - state: icon-stun
+  - type: ItemBorgModule
+    items:
+    - WeaponBorgDisabler
+    - WeaponborgTaser
```

Comment:
```suggestion
```

Reply:
why do you want the taser removed? the disabler and taser are functionally similar, just different projectiles.